app/controllers/declaration/InlandTransportDetailsController.scala

```diff
@@ -41,20 +40,21 @@ class InlandTransportDetailsController @Inject()(
 )(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport with ModelCacheable {
 
-  import forms.declaration.WarehouseDetails._
+  import forms.declaration.InlandModeOfTransportCode._
 
   def displayPage(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
-    request.cacheModel.locations.warehouseIdentification match {
+    request.cacheModel.locations.inlandModeOfTransportCode match {
       case Some(data) => Ok(inlandTransportDetailsPage(mode, form().fill(data)))
       case _          => Ok(inlandTransportDetailsPage(mode, form()))
     }
   }
 
   def submit(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
-    form()
+    InlandModeOfTransportCode
```

Comment:
I think you're importing everything and using form without object above. This is unnecessary and can make wrong thinking about form in 47 line. Is it the same?